pages/linux/compgen.md

```diff
@@ -0,0 +1,23 @@
+# compgen
+
+> Programmable Completion Builtins.
```

Comment:
Can we reword this a bit ? I would want to explain in the tone of what this tool does, rather than a terse one-liner. Something like - `Tool used to ...`